src/solution.c

```diff
@@ -746,6 +771,11 @@ void solution_setup()
 
   SETTING("sbp", "obs_msg_max_size", msg_obs_max_size, TYPE_INT);
 
```

Comment:
I'd like to make sure that these names are clear as above.  I'd rather see gpgsv_output_divisor or something as the naming pattern.